README.md

```diff
@@ -372,7 +356,11 @@ Pubblica Amministrazione con un'unica Identità Digitale da computer, tablet e s
 - [cieid-android-sdk](https://github.com/italia/cieid-android-sdk)
   <img align="right" src="https://img.shields.io/github/stars/italia/cieid-android-sdk?label=%E2%AD%90%EF%B8%8F&logo=github" alt="GitHub stars">
   <img align="right" src="https://img.shields.io/github/issues/italia/cieid-android-sdk" alt="GitHub issues">
-  
+  Android SDK for CIE
```

Comment:
```suggestion
  iOS SDK for CIE
```

Reply:
@bfabio this is the description for cieid-android-sdk, iOS SDK is below

Reply:
@astagi oops, sorry. I got misleaded by my own MD formatting 😅

Reply:
no problem @bfabio :)